ProjectManagerServer/Sources/App/configure.swift

```diff
@@ -1,10 +1,23 @@
 import Vapor
+import Fluent
+import FluentPostgresDriver
 
 // configures your application
 public func configure(_ app: Application) throws {
     // uncomment to serve files from /Public folder
     // app.middleware.use(FileMiddleware(publicDirectory: app.directory.publicDirectory))
-
+    
+    if let databaseURL = Environment.get("DATABASE_URL"), var postgresConfig = PostgresConfiguration(url: databaseURL) {
```

Comment:
3번 질문에 대한 답변)
아실수도 있겠지만 `"DATABASE_URL"` 은 Heroku 에서 설정해주는 PostgreSQL database 에 접근할 수 있는 url 이라고 하네요. 그래서 url 로 configuration 을 설정하는 경우와 그렇지 못한 경우로 나뉘어져 있는 걸로 보여요. (DATABASE_URL 는 postgre sql의 환경변수 이름이라고 합니다. )
그래서 이 환경변수로 접근하지 못하는 경우, 직접 데이터 베이스 host, 유저 이름 등으로 데이터베이스에 접근하는 방식도 있어요 그 방식을 서술해준거라고 보여요. 
다른 접근 방식을 제공하지 않을거라면 else 문을 제거해도 상관 없을 것 같고요, 다른 방식으로 접근하도록 구현할거라면 그대로 놓으셔도 무방할 것 같습니다.

Reply:
> 3번 질문에 대한 답변)
> 아실수도 있겠지만 `"DATABASE_URL"` 은 Heroku 에서 설정해주는 PostgreSQL database 에 접근할 수 있는 url 이라고 하네요. 그래서 url 로 configuration 을 설정하는 경우와 그렇지 못한 경우로 나뉘어져 있는 걸로 보여요. (DATABASE_URL 는 postgre sql의 환경변수 이름이라고 합니다. ) (DATABASE_URL here will represent out postgres database. NEVER hard code the static url from this, heroku will rotate it and it will break your application.)
> 그래서 이 환경변수로 접근하지 못하는 경우, 직접 데이터 베이스 host, 유저 이름 등으로 데이터베이스에 접근하는 방식도 있어요 그 방식을 서술해준거라고 보여요.
> 다른 접근 방식을 제공하지 않을거라면 else 문을 제거해도 상관 없을 것 같고요, 다른 방식으로 접근하도록 구현할거라면 그대로 놓으셔도 무방할 것 같습니다.

Reply:
+ 덧) 데이터 베이스를 heroku 에서 설정하는 거 말고 aws 나 로컬에서 구축했다면 else 처리 방식으로 할 수 있답니다. 짧은 저희 경험에 의하면 저는 저 방식으로 데이터베이스 클라이언트 프로그램을 연결해보았어요..ㅎㅎ